CMakeLists.txt

```diff
@@ -37,6 +37,9 @@ if(BUILD_SHARED_LIBS)
   set(CMAKE_POSITION_INDEPENDENT_CODE YES)
 endif()
 
+## Uncomment line below to include code that provides Secure Enclave support
+##add_compile_definitions(INCLUDE_SECURE_ENCLAVE_SUPPORT)
```

Comment:
@etcwilde `add_compile_definitions` seems to work but is there a better way to set `INCLUDE_SECURE_ENCLAVE_SUPPORT`?